framework/test_app/runners/k8s/k8s_xds_client_runner.py

```diff
@@ -191,6 +193,7 @@ def run(  # pylint: disable=arguments-differ
             config_mesh=config_mesh,
             generate_mesh_id=generate_mesh_id,
             print_response=print_response,
+            is_trusted_xds_server_experimental=is_trusted_xds_server_experimental,
```

Comment:
This won't be needed one it's moved to `ClientDeploymentArgs`. The template will stay the same.